### ISSUE TITLE
New version does not transform properly

### ISSUE THREAD

**Issue by dhaspden** [jacklam718/react-native-popup-dialog#59]
Hello, thanks for the great project.

Just upgraded to the new version and I am getting an error while transforming the files.  Not sure what may have changed but I was upgrading from the previous version (0.9.32).  For the record I was updating to 0.9.33.

I got this error...

```
Bundling `index.ios.js`  98.0% (869/878), failed.
error: bundling failed: "TransformError: /Users/user/Documents/code/project/mobile/
node_modules/react-native-popup-dialog/dist/PopupDialog.js: Unknown plugin 
\"flow-react-proptypes\" specified in \"/Users/user/Documents/code/project/
mobile/node_modules/react-native-popup-dialog/.babelrc.env.development\" at 0, 
attempted to resolve relative to \"/Users/user/Documents/code/project/mobile/
node_modules/react-native-popup-dialog\""
```

I'm not sure what the issue is but I could tackle it if you want (if you can reproduce it.) All I did was run `yarn add react-native-popup-dialog@0.9.33` and it didn't work.

**Comment by jacklam718** [comment]
@dhaspden 
I tried but I can not reproduce the issue.
Thank you for report the issue.

**Comment by jacklam718** [comment]
@dhaspden Thank you for report the issue. I Fixed in version 0.9.34
